<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgGestionCasas</class>
 <widget class="QWidget" name="dlgGestionCasas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1018</width>
    <height>676</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Casas</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../icons.qrc">
    <normaloff>:/icons/icons/christianity.png</normaloff>:/icons/icons/christianity.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Tipo</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QRadioButton" name="rbTodas">
            <property name="text">
             <string>Todas</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rbMasculinas">
            <property name="text">
             <string>Masculinas</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rbFemeninas">
            <property name="text">
             <string>Femeninas</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Provincias</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btAnadirProvincia">
         <property name="text">
          <string>Añadir</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btQuitarProvincia">
         <property name="text">
          <string>Quitar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btQuitarTodasProvincias">
         <property name="text">
          <string>Quitar todas</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListWidget" name="lwProvincias"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Tipos</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cbTipos"/>
       </item>
       <item>
        <widget class="QPushButton" name="btAnadirTipo">
         <property name="text">
          <string>Añadir</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btQuitarTipo">
         <property name="text">
          <string>Quitar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btQuitarTiposTodos">
         <property name="text">
          <string>Quitar todos</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListWidget" name="lwTipos"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtFiltro"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="twCasas"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btModificar">
       <property name="text">
        <string>&amp;Modificar casa</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btResetearFiltros">
       <property name="text">
        <string>Resetear filtros</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>328</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btCerrar">
       <property name="text">
        <string>&amp;Cerrar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections/>
</ui>
